### DISCUSSION TITLE
couple of questions

### DISCUSSION THREAD

**Discussion by RodMyers** [RaspAP/raspap-webgui#1564] [Q&A]
I have RaspAP up and running, using openvpn and IPVanish.  It works good

1. why switch to Wireguard, as I see wireguard has a killswitch option for the "Insiders". I have no issue contributing if necessary.

2. is there an openvpn killswitch option.

3. what about having the web page capthcas. is there an option to work with or around the need to log in via RaspAP.

Thank you

**Comment by billz** [comment]
1. Your choice. Our docs describe it well enough as does wireguard.com
2. No, if there was the docs would mention it. Alternative is google iptables killswitch rules. 
3. Captchas?
